.golangci.yaml

```diff
@@ -1,29 +1,140 @@
+# Based on recommendations in https://olegk.dev/go-linters-configuration-the-right-version
+
 run:
+  # linter execution
   timeout: 30m
+  issues-exit-code: 1
+  concurrency: 4
+  # check tests as well
+  tests: true
```

Comment:
I don't think this is desirable (and especially in the current case where e2e tests are defined as Go tests).
When you want to run linter, you just want to run linter.
Tests is a different target.

Reply:
Perhaps there's some confusion.
Setting `tests: true` runs Go **linters** on the test files, and does **not** run the tests themselves.
Since it's Go code, I see no reason not to lint them.

Reply:
My bad :)